tests/test_async_retry.py

```diff
@@ -243,7 +243,8 @@ async def test_async_transaction_interface_errors(self):
                     async with tx:
                         pass
 
-        with self.assertRaisesRegex(edgedb.InterfaceError, r".*is borrowed.*"):
-            async for tx in self.con.transaction():
-                async with tx:
-                    await self.con.execute("SELECT 123")
+        # with self.assertRaisesRegex(edgedb.InterfaceError,
```

Comment:
nit: let's make sure we remove this code in later commits -- we shouldn't have commented out code in the master branch.